pyuvdata/utils.py

```diff
@@ -169,6 +169,63 @@ def _fits_indexhdus(hdulist):
     return tablenames
 
 
+def _get_fits_extra_keywords(header, keywords_to_skip=None):
+    """
+    Get any extra keywords and return as dict.
+
+    Parameters
+    ----------
+    header : FITS header object
+        header object to get extra_keywords from.
+    keywords_to_skip : list of str
+        list of keywords to not include in extra keywords in addition to standard
+        FITS keywords.
+
+    Returns
+    -------
+    dict
+        dict of extra keywords.
+    """
+    # List standard FITS header items that are still should not be included in
+    # extra_keywords
+    std_fits_substrings = [
+        "HISTORY",
+        "SIMPLE",
+        "BITPIX",
+        "EXTEND",
+        "BLOCKED",
+        "GROUPS",
+        "PCOUNT",
+        "BSCALE",
+        "BZERO",
+        "NAXIS",
+        "PTYPE",
+        "PSCAL",
+        "PZERO",
+        "CTYPE",
+        "CRVAL",
+        "CRPIX",
+        "CDELT",
+        "CROTA",
+        "CUNIT",
+    ]
+
+    if keywords_to_skip is not None:
+        std_fits_substrings.extend(keywords_to_skip)
+
+    extra_keywords = {}
+    # find all the other header items and keep them as extra_keywords
+    for key in header:
+        if np.any([sub in key for sub in std_fits_substrings]):
+            continue
```

Comment:
Is this so if someone has `HISTORY_2_3_MINE"  in their header it won't get put into the fits keywords? Is there also a large precedent for this? I suppose at first glance I would have expected the logic to look like
```
if key in std_fits_substrings:
            continue
```

Reply:
No. It's because NAXIS and the P & C keywords in the list are followed by various integers. I can do something fancier if you think it's needed.

Reply:
Ahh okay that actually makes a lot of sense. I think I had just forgotten about how fits does that.  Maybe add a comment so we (I) don't forget again?

Reply:
sure, I'll add a comment.